app/models/like.rb

```diff
@@ -1,4 +1,4 @@
 class Like < ApplicationRecord
-  belongs_to :CreatedAT
-  belongs_to :UpdateAt
+  belongs_to :author, class_name: 'User'
+  belongs_to :post, counter_cache: :likesCounter
 end
```

Comment:
- [ ] As pointed out by the previous reviewer, please kindly include a method that updates the likes counter.